runtime/lua/vim/treesitter/_meta/misc.lua

```diff
@@ -24,6 +24,11 @@ error('Cannot require a meta file')
 ---@class TSLangInfo
 ---@field fields string[]
 ---@field symbols table<string,boolean>
+---@field major_version? integer
+---@field minor_version? integer
+---@field patch_version? integer
+---@field state_count integer
```

Comment:
Are these fields documented anywhere? Else it might be helpful to add descriptions.

Reply:
Added some brief explanation. Version should be obvious, while state count and supertypes are "expert information" -- either you know what they mean or the information is meaningless anyway.

(And the treesitter docs are in sore need of overhaul, but that has to wait until we are closer to being stable.)